docs/web-vitals.md

```diff
@@ -0,0 +1,362 @@
+# Web Vitals
+
+By default, Honeycomb includes events for all web vitals except first-input-delay. First-input-delay is set to be replaced by interaction-to-next-paint as a core web vital in March 2024.
```

Comment:
Can we add a link here for reference? Ideally it won't be long-lived but if the date moves and/or folks are still used to that first-input-delay, we can avoid confusion a bit.

Reply:
100%— added a link but also dropping a screenshot here for posterity 

![Screenshot 2024-02-13 at 9 26 50 AM](https://github.com/honeycombio/honeycomb-opentelemetry-web/assets/14351110/7c08f9da-afc1-4c5d-8fe7-ffc3fe9fc5b6)